.c9/runners/pru0c.run

```diff
@@ -4,12 +4,13 @@
     "cmd" : [
         "make",
         "-f",
-        "/var/lib/cloud9/extras/pru/Makefile",
+        "$file_path/Makefile",
         "TARGET=$file_base_name",
-        "PRUN=pru0",
+        "PRUN=0",
+        "MODEL=BLACK",
```

Comment:
Why is this hardcoded?

Reply:
It shouldn't be.  I was playing around and didn't clean up.